docs/source/data/data_catalog.md

```diff
@@ -574,8 +574,8 @@ io.list()
 
 You can save data using an API similar to that used to load data.
 
-```eval_rst
-.. caution::  This use is not recommended unless you are prototyping in notebooks.
+```{warning}
```

Comment:
From documentation contributor guidelines: "Prefer to use `note`, `warning` and `important` only, rather than a number of different colours/types of callout."